### PR TITLE
rename to knative.dev/eventing-autoscaler-keda

### PR DESCRIPTION
**Pull request by zroubalik**
Post transfer renaming and chores

Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ gofmt:
 build: gofmt
 	rm -rf ./build/*
 	mkdir -p ./build
-	$(GO_BUILD_VARS) go build -mod vendor -o build/autoscaler-keda cmd/controller/main.go 
+	$(GO_BUILD_VARS) go build -mod vendor -o build/eveting-autoscaler-keda cmd/controller/main.go
```
Comment (markusthoemmes) [inline Makefile:32]:
```suggestion
	$(GO_BUILD_VARS) go build -mod vendor -o build/eventing-autoscaler-keda cmd/controller/main.go 
```

Comment (zroubalik) [reply Makefile:32]: Good catch! Thanks Markus

**Comment by zroubalik** [comment]
@n3wscott can I get the /lgtm again pls? (I just fixed the typo spotted by Markus)

**Pull request merged**